src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -305,4 +305,26 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "13.ea.23-open", _))
   }
+
+  @ChangeSet(order = "144", id = "144-add_graalvm_19_0_2", author = "wololock")
+  def migrate144(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        candidate = "java",
+        version = "19.0.2-grl",
+        url = "https://github.com/oracle/graal/releases/download/vm-19.0.2/graalvm-ce-linux-amd64-19.0.2.tar.gz",
+        platform = Linux64),
```

Comment:
We now require a vendor to be set for all Java versions. You can add an extra param to version of `vendor = Some(Graal)`.

Reply:
Thanks, @marc0der, for explaining that! Pushed changes including the vendor.

Reply:
Awesome, thanks!